docs/swarm/join-nodes.md

```diff
@@ -0,0 +1,131 @@
+<!--[metadata]>
++++
+title = "Join nodes to a swarm"
+description = "Add worker and manager nodes to a swarm"
+keywords = ["guide, swarm mode, node"]
+advisory = "rc"
+[menu.main]
+identifier="join-nodes-guide"
+parent="engine_swarm"
+weight=13
++++
+<![end-metadata]-->
+
+# Join nodes to a swarm
+
+When you first create a swarm, you place a single Docker Engine (Engine) into
+swarm mode. To take full advantage of swarm mode you can add nodes to the swarm:
+
+* Adding worker nodes increase capacity. When you deploy a service to a swarm,
+the Engine schedules the tasks on worker nodes. When you add workers to your
+swarm, you increase the scale of the swarm to handle tasks.
+* Manager nodes increase fault-tolerance. Manager nodes perform the
+orchestration and cluster management functions for the swarm. Among manager
+nodes, a single leader node conducts orchestration tasks. If a leader node
+goes down, the remainin manager nodes will elect a new leader and resume
+orchestration and maintenance of the swarm state.
+
+Before you add nodes to a swarm you must install Docker Engine 1.12 or later on
+the host machine.
+
+See "View swarm configurations" on [Run Docker Engine in swarm mode](swarm-mode.md/#view-swarm-configurations) for more information.
+
+
+## Join as a worker node
+
+To retrieve the join command including the join token for worker nodes, run the
+following command on a manager node:
+
+```bash
+$ docker swarm join-token worker
+
+To add a worker to this swarm, run the following command:
+    docker swarm join \
+    --token SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c \
+    192.168.99.100:2377
+```
+
+Run the command from the output on the worker to join the swarm:
+
+```
+$ docker swarm join \
+  --token SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c \
+  192.168.99.100:2377
+
+This node joined a swarm as a worker.
+```
+
+The `docker swarm join` command does the following:
+
+* switches the Docker Engine on the current node into swarm mode.
+* generates a certificate sigining request to the configured root CA.
+* names the node for the machine hostname
+* joins the current node to the swarm at the manager listen address based
+upon the node acceptance policy and the shared secret.
```

Comment:
I see that you updated the docs using `join-tokens`, thus this part is outdated I think.